test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
yes, i understand that. what i am squeamish about is exposing the implementation of `contains` in a test like this (same problem for `take`). until now dispatch has been of similar function types. e.g. anything that implements functor interface (`(a -> b) -> f a -> f b`), we explicity support.